site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -9,6 +9,80 @@ docs: "DOCS-1419"
 
 This topic describes possible issues users might encounter when using NGINX Gateway Fabric. When possible, suggested workarounds are provided.
 
+### General Troubleshooting
+
+When attempting to diagnose a problem or get support, there are a few important data points that can be collected to help with understanding what issues may exist.
```

Comment:
```suggestion
When investigating a problem or requesting help, there are important data points that can be collected to help understand what issues may exist.
```
Shortening the sentence. Support has specific implications so I favour avoiding the word unless it's in relation to an official support channel.